src/docs/sdk/features.mdx

```diff
@@ -105,6 +105,23 @@ Envelope item:
  * [Java](https://github.com/getsentry/sentry-java/blob/b2e8dd43ea2ef37201b215ec409655e49e3231c8/sentry/src/main/java/io/sentry/SentryEnvelopeItem.java#L131-L155)
  * [C#](https://github.com/getsentry/sentry-dotnet/blob/3bc6c66a14e653982a6a25df033d68d8ad12c4af/src/Sentry/Protocol/Envelopes/EnvelopeItem.cs#L170-L178)
  
+## Attachments
+
+Attachments are files stored alongside an event or transaction. To send an attachment, add it as an [envelope item](/sdk/envelopes/#attachment)
```

Comment:
Can you link an attachment to a transaction?

Reply:
> event or transaction

This reads weird as atm transactions are also events.

Reply:
@bruno-garcia, yes, you can add attachments to transactions. We add them in [Java](https://github.com/getsentry/sentry-java/blob/931464f007ed852548c3b671311dc98d113df960/sentry/src/main/java/io/sentry/SentryClient.java#L359-L360).
In Python, we also add them to transactions if I recall correctly. @mitsuhiko added [`add_to_transaction`](https://github.com/getsentry/sentry-python/blob/de54b4f99bf9bf746d75f48f2a63a27a2cd6eec2/sentry_sdk/attachments.py#L18) in Python, but I don't think we are checking that somewhere: https://github.com/getsentry/sentry-python/search?q=add_to_transactions.

Maybe we should also add `add_to_transaction` to Java, as reading a large attachment for transactions might cause a huge overhead. If so, we should document that here too.

Reply:
We talked on a call and agreed on so just writing down here for the record:
None of us (mobile team) understand in what case it would make sense to add an attachment to a transaction. Better keep it out until we understand the trade offs.

Reply:
video replays

Reply:
Thanks for clarifying @dcramer

Reply:
@bruno-garcia, shouldn't we rather add `add_to_transaction` and set the default to `false`?

Reply:
@philipphofmann Sounds good.

Reply:
@bruno-garcia I added two issues to add this flag to [Java](https://github.com/getsentry/sentry-java/issues/1185) and [.NET](https://github.com/getsentry/sentry-dotnet/issues/750).